Apple/__main__.py

```diff
@@ -312,11 +312,18 @@ def unpack_deps(
     On iOS, as a safety mechanism, any dynamic libraries will be purged from
     the unpacked dependencies.
     """
+    # To create new builds of these dependencies, usually all that's necessary
+    # is to push a tag to the cpython-apple-source-deps repository, and GitHub
```

Comment:
Looks like for this one it's more of pushing a simple Makefile update rather than a tag.

Reply:
It will actually work on a tag; the Makefile does encode the "default" for the purposes of other CI checks, but a tag is sufficient to make the release. 

Of course, a PR doesn't *hurt*, either.